### DISCUSSION TITLE
Would it be possible to have this in a project alongside compose for desktop?

### DISCUSSION THREAD

**Discussion by ajsnarr98** [JakeWharton/mosaic#76] [General]
Would it be possible to have this in a project alongside compose for desktop? Then use compose multiplatform when there is a display present, and this project when there is no display outside the command line?

**Comment by JakeWharton** [comment]
Yes, absolutely.

The first thing to do would be to figure out the code which detects if a GUI display is available. Once you figure that out, it should be a simple `if`/`else` on that value as to whether you call the entrypoint to Compose Desktop or the `runMosaic` function.
